packages/mui-material/src/Alert/Alert.js

```diff
@@ -47,7 +47,6 @@ const AlertRoot = styled(Paper, {
 
   return {
     ...theme.typography.body2,
-    borderRadius: theme.shape.borderRadius,
```

Comment:
What if user passes a custom theme (with a custom border radius property inside it)?

Reply:
Then it will get applied in `Paper` component which is the base component of `Alert`.